### PR TITLE
Merging prestashop tab into Configuration on channel #7837

### PR DESCRIPTION
**Pull request by vishu1993**

### CODE REVIEW

**Comment by vishu1993** [comment]
prestashop1
prestashop2
prestashop4

File: view/sale_channel_form.xml [hunk 1]
```diff
@@ -1,43 +1,46 @@
...
+            <label name="last_prestashop_order_import_time"/>
+            <field name="last_prestashop_order_import_time"/>
+        </group>
+    </xpath>
+    <xpath expr="/form/notebook/page[@id='configuration']" position="inside">
+        <group id="prestashop_buttons" col="4" colspan="4" states="{'invisible': Not(Eval('source') == 'prestashop')}">
+            <button name="test_prestashop_connection" string="Test Prestashop Connection" colspan="4"/>
             <button name="import_prestashop_languages" string="Import Prestashop Languages"/>
             <button name="import_prestashop_order_states" string="Import Prestashop Order States"/>
+            <button name="import_prestashop_orders" string="Import Prestashop Orders"/>
```
Comment (priyankarani) [inline view/sale_channel_form.xml]: buttons should be inside import / export tab

File: view/sale_channel_form.xml [hunk 2]
```diff
@@ -1,43 +1,42 @@
...
         </page>
     </xpath>
-    <xpath expr="/form[@string='Sale Channel']" position="inside">
-        <group id="prestashop_buttons" col="4" colspan="8"
-            states="{'invisible': Not(Eval('source') == 'prestashop')}">
-            <button name="test_prestashop_connection" string="Test Prestashop Connection" colspan="8"/>
+    <xpath expr="/form/notebook/page[@id='configuration']/notebook/page[@id='import_export']/group[@id='channel_common_buttons']" position="before" >
+        <group id="prestashop_last_import_export_time" states="{'invisible': Not(Eval('source') == 'prestashop')}">
+            <label name="last_prestashop_order_import_time"/>
+            <field name="last_prestashop_order_import_time"/>
```
Comment (tarunbhardwaj) [inline view/sale_channel_form.xml]: export should be there https://github.com/openlabs/trytond-prestashop/blob/develop/channel.py#L60

File: view/sale_channel_form.xml [hunk 3]
```diff
@@ -1,43 +1,42 @@
...
+        </group>
+        <group id="prestashop_buttons" col="4" colspan="4" states="{'invisible': Not(Eval('source') == 'prestashop')}">
+            <button name="test_prestashop_connection" string="Test Prestashop Connection" colspan="4"/>
             <button name="import_prestashop_languages" string="Import Prestashop Languages"/>
             <button name="import_prestashop_order_states" string="Import Prestashop Order States"/>
-            <button name="export_prestashop_orders" string="Export Prestashop Orders"/>
+            <button name="import_prestashop_orders" string="Import Prestashop Orders"/>
```
Comment (tarunbhardwaj) [inline view/sale_channel_form.xml]: do we really need these buttons ?

**Pull request merged**